### ISSUE TITLE
Labels are not applied when using the context pad right away

### ISSUE THREAD

**Issue by volkergersabeck** [camunda/camunda-modeler#1502]
__Steps to Reproduce__

1. add e.g. a task and give it a label without closing the editing mode
2. append another task via the context pad
3. entered text is not applied as label of first task

![labels not applied when using context pad](https://user-images.githubusercontent.com/232571/64630238-f6f14a00-d3f4-11e9-8823-d0c4bbd32a8b.gif)

__Expected Behavior__

Labels get applied when blurring the text input

__Environment__

 - OS: MacOS
 - Camunda Modeler Version: 3.3.4

**Comment by volkergersabeck** [comment]
also happens in Cawemo after we upgraded to latest bpmn-js

**Comment by pinussilvestrus** [comment]
Seems like it stopped working with version [`v3.2.0`](https://github.com/camunda/camunda-modeler/blob/develop/CHANGELOG.md#320), before, it worked fine.

**Comment by barmac** [comment]
Closed via https://github.com/camunda/camunda-modeler/pull/1513#pullrequestreview-293041737

**Comment by hughgallagher** [comment]
This is still happening for me on Cawemo. Labels are not appearing. Is Cawemo not using the latest js??? How do I fix this?

**Comment by linus-amg** [comment]
@hughgallagher Cawemo is using version 5.1.0 of bpmn-js, I was unable to reproduce the issue which you seem to face. Could you write down the exact steps you make to reproduce the issue?

**Comment by hughgallagher** [comment]
I am not sure what it was that triggered it. I was using cawemo for about an hour and then it just stopped displaying labels I was creating on tasks, events etc. exactly as in the OP. However, when I uploaded the bpmn file to bpmn.io it worked fine. I did some work and then exported from bpmn.io and imported into cawemo overwriting the diagram that had the problem above and the same diagram no longer had the issue!

Using latest Chrome on Windows 10:-1:

Chrome Version 78.0.3904.70 (Official Build) (64-bit)
